src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/WcfDuplexService.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ServiceModel;
+using System.Threading.Tasks;
+
+namespace WcfService
+{
+    [ServiceBehavior(AddressFilterMode = AddressFilterMode.Any)]  
+    public class WcfDuplexService : IWcfDuplexService, IWcfDuplexService_DataContract, IWcfDuplexService_Xml
```

Comment:
Not required for this checkin, but I think it would be easier to maintain if we had a separate folder for services (testservices?) like you do for testhosts.  We can discuss or rearrange after checkin.  I'm thinking something like App_code\testtypes, App_code\testservices, and App_Code\testhosts.